tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -165,6 +174,11 @@ func handler(ctx context.Context, ev map[string]interface{}) error {
 		return processCWEvent(ctx, evt)
 	case *events.KinesisEvent:
 		return processKinesisEvent(ctx, evt)
+	case *events.SQSEvent:
+		return processSQSEvent(ctx, evt)
+	//When setting up S3Notification to SQS, a test event is first sent to the queue
```

Comment:
is this behaviour documented somewhere?

Reply:
Yes [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html#:~:text=Example%20messages,following%20test%20message.)

Reply:
can we include that link in this comment please